velox/connectors/Connector.h

```diff
@@ -331,6 +338,7 @@ class ConnectorQueryCtx {
   memory::MemoryPool* const operatorPool_;
   memory::MemoryPool* const connectorPool_;
   const Config* const sessionProperties_;
+  const common::Identity* identity_;
```

Comment:
Storing raw pointers is not safe. Can we store this by value or use a smart pointer?

Reply:
I think it's not a big problem since `ConnectorQueryCtx` is always outlived by `QueryCtx`.  This way it's more clear that `QueryCtx` owns the object and `ConnectorQueryCtx` is just borrowing it.